code/solr/services/SolrService.php

```diff
@@ -1,30 +1,29 @@
 <?php
 
-Solr::include_client_api();
+namespace SilverStripe\FullTextSearch\Solr\Services;
 
-/**
- * The API for accessing a specific core of a Solr server. Exactly the same as Apache_Solr_Service for now.
- */
-class SolrService_Core extends Apache_Solr_Service
-{
-}
+use SilverStripe\Core\Config\Config;
+use SilverStripe\FullTextSearch\Solr\Solr;
+use Silverstripe\Core\ClassInfo;
 
+Solr::include_client_api();
```

Comment:
We should probably tidy this up a bit now (incl. the contents of this method). If we can't include the thirdparty lib with composer then we could add it to the composer autoloader instead, which would remove the need for this type of logic